daemon/graphdriver/devmapper/deviceset.go

```diff
@@ -2761,6 +2761,14 @@ func NewDeviceSet(root string, doInit bool, options []string, uidMaps, gidMaps [
 				return nil, errors.New("dm.thinp_autoextend_threshold must be greater than 0 and less than 100")
 			}
 			lvmSetupConfig.AutoExtendThreshold = per
+		case "dm.libdm_log_level":
+			level, err := strconv.ParseUint(val, 10, 32)
```

Comment:
Considering that `DmLogLevel(value int)` does bounds checking, we could just parse a 64bit end here and defer bounds errors to `DmLogLevel(value int)`.